### PR TITLE
Update docs about recursive wildcards in strings

### PR DESCRIPTION
**Pull request by jedi22**
The docs need to be updated to make it clear you cannot recursively wildcard in the middle of a string.

### CODE REVIEW

**Comment by osqueryer** [comment]
:-1: The commit f441553d0cbe73723cbf359a8c600e984e84c369 (Job results: [981](https://jenkins.osquery.io/job/osqueryPullRequestOSX/981/)) failed one or more tests (**macOS/OS X**).

**Comment by osqueryer** [comment]
:-1: The commit f441553d0cbe73723cbf359a8c600e984e84c369 (Job results: [982](https://jenkins.osquery.io/job/osqueryPullRequestOSX/982/)) failed one or more tests (**macOS/OS X**).

**Comment by jedi22** [comment]
Address #3213

**Comment by jedi22** [comment]
ok to test

File: docs/wiki/deployment/file-integrity-monitoring.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ For example, you may want to monitor `/etc` along with other files on a Linux sy
 
 The two areas below that are relevant to FIM are the scheduled query against `file_events` and the added `file_paths` section. The `file_events` query is scheduled to collect all of the FIM events that have occurred on any files within the paths specified within `file_paths` on a five minute interval. At a high level this means events are buffered within osquery and sent to the configured _logger_ every five minutes.
 
+*Note*: You cannot match recursively inside a path. For example `/Users/%%/Configuration.conf` is not a valid wildcard.
```
Comment (theopolis) [inline docs/wiki/deployment/file-integrity-monitoring.md]: I'd recommend **Note:**

Comment (jedi22) [inline docs/wiki/deployment/file-integrity-monitoring.md]: My markdown fu is weak

**Pull request merged**